docs/cinder-csi-plugin/using-cinder-csi-plugin.md

```diff
@@ -133,7 +133,21 @@ cinder.csi.openstack.org   2019-07-29T09:02:40Z
 
 ```
 
-> NOTE: If using certs(`ca-file`), make sure to update the manifests (controller and node plugin) to mount the location of certs as volume onto container as well.
+> NOTE: If using certs(`ca-file`), make sure to add the additional mount to the manifests (controller and node plugin) to mount the location of certs as volume onto container. For example, add `ca-cert` in `/etc/cacert` folder:
+
+```
+       volumeMounts:
```

Comment:
how about we add in manifests itself with commented lines. It can say enable as per requirement

Reply:
right this is better option, will update

Reply:
@ramineni updated with commented sections